messagebus/src/vespa/messagebus/network/rpcnetwork.cpp

```diff
@@ -382,10 +382,11 @@ RPCNetwork::sync()
 void
 RPCNetwork::shutdown()
 {
-    _executor->shutdown();
     _executor->sync();
     _transport->ShutDown(false);
     _threadPool->Close();
+    _executor->shutdown();
```

Comment:
if moving executor shutdown after network shutdown is needed, then the first executor sync is probably just extra waiting...

Reply:
I will keep it there so we get rid of outgoing stuff before closing down network.